test/jdk/javax/swing/JButton/SwingButtonResizeTestWithOpenGL.java

```diff
@@ -209,10 +210,7 @@ private BufferedImage getButtonImage() {
     }
 
     private void disposeFrame() {
-        if (frame != null) {
-            frame.dispose();
-            frame = null;
-        }
+        frame.dispose();
```

Comment:
null check might be required here.

Reply:
> null check might be required here.

But we are not setting it as null anywhere in the code.

Reply:
What if its not initialized at first place?

Reply:
`Frame is assigned null after disposing which is not required and can be removed.` I hope u r referring to this comment. I hope she meant to remove null assignment after dispose is called.

Reply:
Right, I meant remove only the null assignment not the null check.